runtime/arc.js

```diff
@@ -31,6 +31,7 @@ class Arc {
     this.id = id;
     this._nextLocalID = 0;
     this._activeRecipe = new Recipe();
+    this._acceptedSuggestions = [];
```

Comment:
Not sure about calling these suggestions, that seems like a higher level concept.